LQResultProducer.py

```diff
@@ -140,8 +144,8 @@
 #NormalWeightMuMu = str(lumi)+'*weight_central*((pass_HLTMu50+pass_HLTTkMu50)>0)'+doublemuHLT+doubleMuIdScale+doubleMuIsoScale+trackerHIP1+trackerHIP2
 #NormalWeightMuNu = str(lumi)+'*weight_central*((pass_HLTMu50+pass_HLTTkMu50)>0)'+singlemuHLT+singleMuIdScale+singleMuIsoScale+trackerHIP1
 #NormalWeightEMu = str(lumi)+'*weight_central*((pass_HLTMu50+pass_HLTTkMu50)>0)'+singlemuHLTEMU+MuIdScaleEMU+MuIsoScaleEMU+eleRECOScale+eleHEEPScale+trackerHIPEMU
-NormalWeightMuMu = str(lumi)+'*weight_central'+doublemuHLT+doubleMuIdScale+doubleMuIsoScale+trackerHIP1+trackerHIP2+bTagSFmedium
-NormalWeightMuNu = str(lumi)+'*weight_central'+singlemuHLT+singleMuIdScale+singleMuIsoScale+trackerHIP1
+NormalWeightMuMu = str(lumi)+'*weight_central'+doublemuHLT+doubleMuRecoSF+trackerHIP1+trackerHIP2+bTagSFmedium
```

Comment:
why are ID and ISO SF not here?

Reply:
I thought (after a re-read, and seeing no SF table) the recommendation was that they need not be applied, but I think the phrasing is a little confusing (e.g., "No additional treatment needed").
https://twiki.cern.ch/twiki/bin/view/CMS/HighPtMuonReferenceRun2#Isolation
https://twiki.cern.ch/twiki/bin/view/CMS/HighPtMuonReferenceRun2#Identification
Also that would contradict: https://twiki.cern.ch/twiki/bin/view/CMS/MuonLegacy2016#ID_and_ISO_efficiencies
I'll add them back in.

Reply:
Hmm ok......let's follow up, a message to the Muon POG hypernews can clear it up quickly, just say what ID and ISO we use, point to the Twiki and say it looks like we should not apply SFs, is this correct?